bin/add-roles.sh

```diff
@@ -18,6 +18,7 @@
 ./bin/utils/ccd-add-role.sh "caseworker-ras-validation"
 ./bin/utils/ccd-add-role.sh "admin-access"
 ./bin/utils/ccd-add-role.sh "full-access"
+./bin/utils/ccd-add-role.sh "hearing-schedule-access"
 ./bin/utils/ccd-add-role.sh "civil-administrator-standard"
 ./bin/utils/ccd-add-role.sh "civil-administrator-basic"
 ./bin/utils/ccd-add-role.sh "hearing-schedule-access"
```

Comment:
Does this role need to be in here twice? I think this is an accidental conflict resolution error :)

Reply:
Thank you! solved